lib/api/expect/_baseExpect.js

```diff
@@ -147,7 +147,7 @@ class ExpectInstance extends EventEmitter {
   }
 
   retryCommand() {
-    const promise = this.command();
+    const promise = this.command(...this.commandArgs);
     this.handleCommandPromise(promise);
```

Comment:
is this calling elsewhere for cookie.js ?

Reply:
Yes, it's being used by `lib/api/expect/assertions/_baseAssertion.js`, and in `expect` API as well. I will update the thread if there is any impact on this change